stock_voucher/views/stock_picking_views.xml

```diff
@@ -18,10 +18,7 @@
         <field name="inherit_id" ref="stock.vpicktree"/>
         <field name="arch" type="xml">
             <field name="name" position="after">
-                <!-- TODO replace for one2mant_tags when migrated -->
-                <!-- we try to use m2m tags but it doenst display ok on tree view -->
-                <field name="vouchers"/>
-                <!-- <field name="voucher_ids" widget="one2many_tags"/> -->
+                <field name="vouchers" widget="one2many_tags"/>
```

Comment:
one2many_tags widget does not exist (it does in web_widget_one2many_tags but this module is a commented dependency so here the widget does not really is references). you can use many2many_tags instead (as mentioned in the example in a early comment in this review)